src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/LSPDocument.cs

```diff
@@ -3,38 +3,30 @@
 
 using System;
 using System.Collections.Generic;
+using Microsoft.CodeAnalysis.Text;
+using Microsoft.VisualStudio.Text;
 
 namespace Microsoft.VisualStudio.LanguageServerClient.Razor
 {
-    /// <summary>
```

Comment:
Y no doc?

Reply:
Too much work to maintain in a currently growing system.